repo2docker_wholetale/wholetale.py

```diff
@@ -49,6 +49,20 @@ def get_build_script_files(self):
         )
         return files
 
+
+    def get_build_scripts(self):
+        return super().get_build_scripts() + [
+            (
+                "root",
+                r"""
+                wget -O /usr/local/bin/reprozip https://github.com/cirss/reprozip-static/releases/download/v1.0.16-r1/reprozip-1.016-linux-x86-64-static \
```

Comment:
Can you convert that hardcoded url for `reprozip-static` into templated env var?

Reply:
Do you mean using `wt_env`?  I gather that the idea is that the reprozip version we use might change independent of our r2d. If so, wouldn't we want to change it globally (i.e., with an update to r2d_wt) instead of per-image (via env)?

Reply:
I figured that it might happen more often than our releases, since it's active work. I wanted to have an option to bump it in prod without going through the hassle of our release. I can be convinced it's a bad idea